platform/linux/.gitconfig

```diff
@@ -0,0 +1,9 @@
+[user]
+	name = Scott Mebberson
+	email = scott@scottmebberson.com
```

Comment:
Hmm is there a way we can have this without specifying the user?

Reply:
Oh right, indeed.

Reply:
I'll have to improve this via the install process. I'll do that shortly.